app/models/application_record.rb

```diff
@@ -4,7 +4,12 @@ class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
 
   def self.lockbox_options
-    { previous_versions: [{ padding: true, master_key: Settings.lockbox.master_key }, { padding: true }] }
+    {
+      previous_versions: [
+        { padding: false },
+        { padding: false, master_key: Settings.lockbox.master_key }
```

Comment:
I inverted these rows in case it fallsback in the order of this array.

Reply:
Yes, smart idea!